tests/test_errors.py

```diff
@@ -280,9 +280,9 @@ def finalize(self, cfg: Any) -> None:
         Expected(
             create=lambda: OmegaConf.create({"foo": "${int.missing}", "int": 0}),
             op=lambda cfg: getattr(cfg, "foo"),
-            exception_type=InterpolationKeyError,
+            exception_type=InterpolationResolutionError,
             msg=(
-                "ConfigKeyError while resolving interpolation: Error trying to access int.missing: "
+                "ConfigTypeError raised while resolving interpolation: Error trying to access int.missing: "
```

Comment:
In base.py there's this try/except block that was triggering the previous exception => the second except should be removed:
```python
        try:
            parent, last_key, value = root_node._select_impl(
                inter_key,
                throw_on_missing=True,
                throw_on_resolution_failure=True,
                memo=memo,
            )
        except MissingMandatoryValue as exc:
            raise InterpolationToMissingValueError(
                f"MissingMandatoryValue while resolving interpolation: {exc}"
            ).with_traceback(sys.exc_info()[2])
        except ConfigKeyError as exc:
            raise InterpolationKeyError(
                f"ConfigKeyError while resolving interpolation: {exc}"
            ).with_traceback(sys.exc_info()[2])
```

Reply:
Yes, this is why the code coverage failed.